plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -5659,19 +5671,21 @@ private void testReadWithPartitionSchemaMismatchByName(Session session, HiveStor
         String tableName = testReadWithPartitionSchemaMismatchAddedColumns(session, format);
 
         // with mapping by name also test behavior with dropping columns
-        // start with table with a, b, c, _part
-        // drop b
-        assertUpdate(session, "ALTER TABLE " + tableName + " DROP COLUMN b");
-        // create new partition
-        assertUpdate(session, "INSERT INTO " + tableName + " values (21, 22, 20)", 1); // a, c, _part
-        assertQuery(session, "SELECT a, c, _part FROM " + tableName, "VALUES (1, null, 0), (11, 13, 10), (21, 22, 20)");
-        assertQuery(session, "SELECT a, _part FROM " + tableName, "VALUES (1,  0), (11, 10), (21, 20)");
-        // add d
-        assertUpdate(session, "ALTER TABLE " + tableName + " ADD COLUMN d bigint");
-        // create new partition
-        assertUpdate(session, "INSERT INTO " + tableName + " values (31, 32, 33, 30)", 1); // a, c, d, _part
-        assertQuery(session, "SELECT a, c, d, _part FROM " + tableName, "VALUES (1, null, null, 0), (11, 13, null, 10), (21, 22, null, 20), (31, 32, 33, 30)");
-        assertQuery(session, "SELECT a, d, _part FROM " + tableName, "VALUES (1, null, 0), (11, null, 10), (21, null, 20), (31, 33, 30)");
+        if (ALLOWED_TO_DROP_TABLE_SERDES.contains(format.getSerde())) {
```

Comment:
Are we invoking this method - for table format which doesn't support dropping column, then I guess we need to fix it out there.

Reply:
Could you clarify what you mean please ? 

I believe that running `testReadWithPartitionSchemaMismatchAddedColumns` should be run with all storage formats.